backend/mobile-backend/schema.graphql

```diff
@@ -0,0 +1,60 @@
+# This "input" configures a global authorization rule to enable public access to
+# all models in this schema. Learn more about authorization rules here: https://docs.amplify.aws/cli/graphql/authorization-rules
+input AMPLIFY { globalAuthRule: AuthRule = { allow: public } } # FOR TESTING ONLY!
+
+type GameSession @model {
+  id: ID! @primaryKey
+  game: Game! @hasOne
+  startTime: AWSDateTime
+  phaseOneTime: Int
+  phaseTwoTime: Int
+  teams: [Team] @hasMany
+  currentQuestion: Int!
+  isPhaseOne: Boolean!
```

Comment:
Initially, we had a game status that was an enum to indicate the state of the game. I think we're missing that here. We should add any new state needed to that enum